tests/test_habitat_quality.py

```diff
@@ -245,6 +246,14 @@ def test_habitat_quality_presence_absence_regression(self):
             # so we should exclude those new nodata pixel values.
             assert_array_sum(raster_path, assert_value, include_nodata=False)
 
+        for csv_filename in ['rarity_c_regression.csv',
+                             'rarity_f_regression.csv']:
+            csv_path = os.path.join(args['workspace_dir'], csv_filename)
+            rarity_table = pandas.read_csv(csv_path)
+            assert 'LULC_code' in rarity_table.columns
+            self.assertAlmostEqual(rarity_table['rarity_value'].sum(),
```

Comment:
Could you leave a comment of where the 0.6667 values comes from?

Reply:
Yes, and I rewrote this part of the test; hopefully it's a bit more meaningful now. (I also switched from pandas to `csv.reader`.)